MapboxNavigation/Style.swift

```diff
@@ -258,6 +258,21 @@ open class StylableLabel: UILabel {
     }
 }
 
+/// :nodoc
+@objc(MBStyleableView)
+open class StylableView: UIView {
```

Comment:
nit: s/Stylable/Styleable or the other way around? Both variations seems to be wildly used but we should be consistent. If we settle for styleable, would you mind renaming the existing stylable classes?

Reply:
“[Stylable](https://en.wiktionary.org/wiki/stylable#English)” is the preferred spelling.